src/client/public/config/openfin/launcher-dev.json

```diff
@@ -39,7 +40,7 @@
     }
   },
   "runtime": {
-    "arguments": "--remote-debugging-port=9222",
+    "arguments": "--remote-debugging-port=9222 --disable-legacy-window",
```

Comment:
Why did we add this switch?

Reply:
In order to make window transparent (referring to this docs https://developers.openfin.co/docs/faq#can-i-make-a-transparent-window-in-openfin) 

![image](https://user-images.githubusercontent.com/1596197/74444441-2ceb9c00-4e75-11ea-9f22-81ba6d6197f4.png)